src/FSM/States/Downloading.cpp

```diff
@@ -58,8 +58,7 @@ namespace sua {
             Logger::info("Bundle version (spec): '{}'", ctx.desiredState.bundleVersion);
             Logger::info("Bundle version (file): '{}'", bundleVersion);
 
-            if(ctx.bundleChecker->isBundleVersionConsistent(
```

Comment:
just a question: is this method not used any more? can it be deleted if the check is performed directly?

Reply:
Removed